src/metpy/calc/thermo.py

```diff
@@ -2574,3 +2574,49 @@ def specific_humidity_from_dewpoint(pressure, dewpoint):
     """
     mixing_ratio = saturation_mixing_ratio(pressure, dewpoint)
     return specific_humidity_from_mixing_ratio(mixing_ratio)
+
+
+@exporter.export
+@preprocess_xarray
+@check_units('[pressure]', '[temperature]', '[temperature]')
+def lifted_index(pressure, temperature, parcel_profile):
+    """Calculate Lifted Index from the pressure temperature and parcel profile.
+
+    Lifted index formula:
+    LI = T500 - Tp500
+    where:
+    T500 is the measured temperature at 500 hPa.
+    Tp500 is the temperature of the lifted parcel at 500 hPa.
+
+    Lifted Index is the mean mixing ratio of the lowest 3000' and the potential temperature of
+    the predicted afternoon high are used. The Lifted Index can be used as a diagnostic tool as
```

Comment:
Something is off in this sentence, maybe "ratio of" should be "ratio if"?

Reply:
This information was copied from issue #632. I didn't include the link in issue #632 where the equation was coming from. I can include this. Is this information that you are looking for?

Reply:
Found a PDF document by Doswell and Schultz 2006 for the origin of most severe indices. Lifted index was created by Galway 1956. Should I include this reference for the lifted index?

Reply:
Well, #632 didn't have "Lifted Index is..." (I guess you added that?), and the combination of that with the other text doesn't make sense. #632 was just copying it (as a placeholder) from https://weather.cod.edu/sirvatka/si.html; the text was implying that as a forecast tool, the mean mixing ratio of the lowest 3000 feet and the potential temperature of the forecasted afternoon high should be used (as an estimate for what the conditions in the mixed convective boundary layer will be). All I know is that we need an intelligible sentence for the docs.

I had no idea tracking down the original reference for Lifted Index would be so easy. Yes, citing and [linking](https://journals.ametsoc.org/doi/pdf/10.1175/1520-0477-37.10.528) to that would be great--so long as our implementation is consistent with what the paper says.

Reply:
I could have modified the text but I wasn't sure if this would have been accepted. Only to say that the lifted index is the subtraction between lifted parcel and measured temperature at 500 hPa. I should have removed some of the text which doesn't reference the equation. Sorry about that. I am still new at this. I will include the references that I have mentioned earlier that support the equation.

Reply:
Changed the description to explain the equation. Also added references to the lifted index and the corresponding links. Hope this satisfy the information that you need.

Reply:
No worries, the updates look great!